exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetRecordWriter.java

```diff
@@ -374,6 +381,13 @@ public void endRecord() throws IOException {
 
   @Override
   public void abort() throws IOException {
+    cleanup();
```

Comment:
Does the new code call abort()? Searched master source and found no references. On the other hand, cleanup() is called from WriterRecordBatch. Should the file system cleanup stuff move there?